examples/contract.rs

```diff
@@ -20,7 +22,9 @@ fn main() {
         .unwrap()
         .confirmations(4)
         .options(Options::with(|opt| {
-            opt.value = Some(5.into())
+            opt.value = Some(5.into());
+            opt.gas_price = Some(5.into());
+            opt.gas = Some(1_000_000.into());
```

Comment:
`gas` is optional in the RPC - backend should figure out the value itself. I suspect that your transaction always reverts hence ganache is not able to estimate how much gas is needed (it reverts no matter how much gas it provides so it eventually fails by reaching block gas limit)